testing/tests/test_var_fps.py

```diff
@@ -71,12 +71,12 @@ def test_classification(self):
                 res = {'score': score_meta, 'score_ul': score_ul, 'score_sl': score_sl}
                 res['master_filename'] = filename
                 res['rendition_type'] = rendition_name
-                res['is_correct'] = not tamper
+                res['is_tamper'] = tamper
                 results.append(res)
         df_res: pd.DataFrame = pd.DataFrame(results)
         df_res.set_index(['master_filename', 'rendition_type'], inplace=True)
         df_res.sort_index(inplace=True)
-        df_res['prediction'] = df_res['score'] <= 0
+        df_res['prediction'] = df_res['score'] > 0
```

Comment:
Isn't -1 = tamper? In that case, this would result in a False prediction when the meta model returns -1.

Reply:
Negative score was "tamper" before, because OCSVM distance function, which has a negative value for an outlier, was used as a score. I think, it makes more sense to have Positive class designate abnormal (much less frequent) state, and model output in (0, 1) interval.